pkg/controller/kieapp/constants/constants.go

```diff
@@ -173,7 +173,7 @@ const (
 	ConsoleAuthoringCPURequests = "1500m"
 	ConsoleAuthoringMemRequests = "3Gi"
 	ConsoleProdCPURequests      = "500m"
-	ConsoleProdMemRequests      = "1.5Gi"
+	ConsoleProdMemRequests      = "1500m"
```

Comment:
shouldn't it be 1500Mi or keep as 1.5Gi?

Reply:
We can be, I was just matching the other memory settings. Your call though

Reply:
done